examples/creatives/base_simid_creative.js

```diff
@@ -69,21 +79,22 @@ class BaseSimidCreative {
    * @param {!Object} eventData Data from the event.
    */
   onInit(eventData) {
+    this.updateInternalOnInit(eventData);
+    this.simidProtocol.resolve(eventData, {});
+  }
+
+  /**
+   * Updates internal data on initialization call.
+   *
+   * Note: When overriding the onInit function and not wishing
+   * to always resolve, subclasses may instead use this function.
+   * @param {!Object} eventData Data from the event.
```

Comment:
`@protected` ?

Reply:
Yeah, I labeled all these functions protected, since they are meant to be overridden.